src/base/seeds/Seed.ts

```diff
@@ -1,6 +1,6 @@
 import * as api from '@app/api';
 import type { Config } from '@app/config';
-import * as proj from '@app/project';
+import * as proj from '@app/base/projects/Project';
```

Comment:
Same comment as below, we shouldn't be importing things between sub-apps (`base/*`), this creates a dependency spaghetti.